atlp-rwanda/e-commerce-ones-and-zeroes-fn
# [finishes #187355039] redux toolkit setup

**What does this PR do??**
This PR is the set up for redux toolkit

**Description of Task to be completed?**
  The setup for redux which will allow us to use state management library to manage state of our react application effectively and predictably.

**Why should we use Redux?**
 [Importance of using Redux toolkit in projects](https://andela.com/blog-posts/why-you-should-switch-to-redux-toolkit-library)

**How should this be manually tested?** 

Step1> Clone The project
Step2> Switch to branch ch-redux-setup
Step3> Install all packages by npm install
Step4> Run the project by npm run start 
Step 5> When the project is running, open the link http://localhost:9000/ to see the project on the browser

**What are the relevant pivotal tracker**

[Pivotal tracker story](https://www.pivotaltracker.com/story/show/187355039)

**Screenshot**
![image](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/125905987/5ec418ce-c2a0-4a9f-af7c-15cda2075c13)
